assets/model_monitoring/components/model_monitor/model_monitor_metric_publisher/spec.yaml

```diff
@@ -0,0 +1,62 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_metric_outputter
+display_name: Model Monitor - Metric Publisher
+description: Azure Monitor Publisher for the computed model monitor metrics.
+version: 0.3.6
+is_deterministic: true
+
+code: ../../src/
+entry:
+  file: ./model_monitor_metric_publisher/run.py
+
+inputs:
+  signal_metrics:
+    type: mltable
+    mode: direct
+  monitor_name:
+    type: string
+  signal_name:
+    type: string
+  location:
```

Comment:
Is there a way we don't need to be location aware for publishing custom metrics?

Reply:
Similar to the workspace_resource_id, there is some env var we can grok this infromation from, namely `Env var AZUREML_SERVICE_ENDPOINT: https://eastus2euap.api.azureml.ms`. But again we do not how sure we can depend on them. Thoughts?

Reply:
Same to the comment above. Updated to extract from the env var.